samples/BenchmarkDotNet.Samples/JIT/Jit_Inlining.cs

```diff
@@ -5,6 +5,9 @@ namespace BenchmarkDotNet.Samples.JIT
 {
     // See http://en.wikipedia.org/wiki/Inline_expansion
     // See http://aakinshin.net/en/blog/dotnet/inlining-and-starg/
+#if !CORE
+    [Diagnostics.Windows.Configs.InliningDiagnoserConfig]
```

Comment:
> By the way, is it possible to implement diagnosers for .NET Core?

We are limited to what API Microsoft/other software vendors are exposing. If MS releases .NET Core ETW lib then there is no problem. I am planning to implement .NET Core Memory Diagnoser next week.